guides/plugins/apps/administration/add-custom-modules.md

```diff
@@ -0,0 +1,142 @@
+# Add custom modules
+
+In your app, you are able to add your own modules to the administration. Your custom modules are loaded as iframes which are embedded in the Shopware administration and within this iframe, your website will be loaded and shown.
+
+In order to create a custom module, you need to define an admin element to define `<admin>` extensions. In there, you can add your module by defining a `<module>` element.
+
+To configure your module you can set it up with with some addtional attributes.
+* `name` (required): The technical name of the module. This is the name your module is referenced with.
+* `source` (optional): The URL to your app servers endpoint from which the module is served from. This can be omitted if you want to define an menu item that should serve as a parent menu item for other app modules.
```

Comment:
```suggestion
* `source` (optional): The URL to your app servers endpoint from which the module is served from. This can be omitted if you want to define a menu item that should serve as a parent menu item for other app modules.
```